src/drawer.canvas.js

```diff
@@ -48,6 +48,25 @@ WaveSurfer.util.extend(WaveSurfer.Drawer.Canvas, {
 
         this.clearWave();
     },
+    
+    updateWidthnHeight: function () {
```

Comment:
First of all, the method name is weird, that n looks like a typo.
Also, could you factor out the height changes into a new `updateHeight` method or something? There's also a different ticket that asks for a updateHeight method (when going fullscreen for example) so this change could then benefit both tickets.